blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 3mdeb open-source contributions
+abstract: 'We are excited to share with you the open-source contributions
+           of our team. We have been actively contributing to a number of
+           open-source projects that have a significant impact on the industry.
+           Through these contributions, we have been able to help improve the
+           overall functionality and security of these projects, while also
+           gaining valuable experience and knowledge that we can apply to our
+           client work.'
+cover: /covers/image-file.png
+author: norbert.kaminski
+layout: post
+published: true
+date: 2023-04-27
+archives: "2023"
+
+tags:
+  - contribution
+  - coreboot
+  - fwupd
+  - trenchboot
+categories:
+  - Manufacturing
+
+---
+
+We are excited to share with you the open-source contributions of our team.
+We have been actively contributing to a number of open-source projects that have
+a significant impact on the industry. Through these contributions, we have been
+able to help improve the overall functionality and security of these projects,
+while also gaining valuable experience and knowledge that we can apply to our
+client work.
+
+In addition to contributing to these open-source projects, our team is also
+passionate about promoting the benefits of open-source software. We believe
+that open-source software offers numerous advantages over proprietary software,
+such as greater transparency, flexibility, and security. By contributing to
+these projects and advocating for open-source software, we aim to foster
+a culture of collaboration, innovation, and inclusivity in the technology
+industry. We are committed to making a positive impact on the community
+through our work, and we look forward to continuing our open-source
+contributions in the future.
+
+## Our contributions:
+
+### coreboot
+
+coreboot is an open-source firmware that provides a lightweight, secure,
+and fast boot experience for PCs, laptops, servers, and embedded devices.
+If you are interested in improving the security of your devices while reducing
+your dependence on proprietary firmware, you could benefit from using coreboot.
+We can also help you reduce time-to-market by simplifying the firmware
+development process.
+
+<!-- List of contribution-->
+
+### fwupd
+
+fwupd is an open-source daemon that manages the firmware updates of various
+devices. You will surely benefit from using fwupd if you are interested in
+automating the firmware update process on your devices and reducing the risk of
+security vulnerabilities. fwupd is compatible with a wide range of devices,
+including laptops, desktops, and IoT devices. Our team can help you seamlessly
+integrate into the fwupd ecosystem across a variety of platforms and operating
+systems. Say goodbye to clunky update processes and hello to a streamlined,
+hassle-free experience with fwupd!
```

Comment:
a/a - you can always link to the project webpage/documentation to follow the DRY rule.